worksheets/rollups/create.http

```diff
@@ -0,0 +1,160 @@
+
+PUT localhost:9200/_opendistro/_rollup/jobs/example
+Content-Type: application/json
+
+{
+  "rollup": {
+    "enabled": true,
+    "schedule": {
+      "interval": {
+        "period": 1,
+        "unit": "Minutes",
+        "start_time": {{$timestamp}}
+      }
+    },
+    "last_updated_time": {{$timestamp}},
```

Comment:
Is anything strange going on here with the formatting for there to be syntax highlighting on some of the `{$timestamp}` and `,`?

Reply:
Github just doesn't read the {{$timestamp}} syntax correctly